tests/conftest.py

```diff
@@ -1,86 +1,23 @@
 import numpy as np
 import pytest
+
 import neurostatslib.basis as basis
 
+
 def pytest_generate_tests(metafunc):
```

Comment:
can we get a docstring describing what this does?

Reply:
right now it is not used right now because I wrote the test using pytest.mark.parametrize instead. I kept it mostly to remember that it exists as an option because you can use to handle complex parametrizations and you can do a lot of cool things with it:
https://docs.pytest.org/en/latest/how-to/parametrize.html#basic-pytest-generate-tests-example

Reply:
oh if it's not used, we should just remove it. I agree that's neat but it will still be in the git history if you need to refer back to it, or you can paste it into an issue or something if you think it will be used in the future.

the docstring is good though!